recipes/_dummy/meta.yaml

```diff
@@ -0,0 +1,2 @@
+package:
+  name: _dummy
```

Comment:
It would be better to have this dummy recipe somewhere else (tmp dir?) -- otherwise the build system may see it as a package that needs to be in the bioconda channel.